src/components/Card/card.js

```diff
@@ -0,0 +1,29 @@
+import CounterBox from "../CounterBox/counterBox";
+import Modal from "react-bootstrap/Modal";
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
import bootstrap at app or index js file, this type of CSS should be load on top level of the app @PuniaMohit